README.md

```diff
@@ -72,26 +77,77 @@ to execute the following step one time.
 GO111MODULE=on go get github.com/jteeuwen/go-bindata/go-bindata@v3.0.8-0.20180305030458-6025e8de665b
 ```
 
-To build the operator, simply:
+#### Clone the repo
 
 ```bash
 git clone https://github.com/istio/operator.git
 cd operator
+```
+
+#### CLI
+
+To build the operator CLI, simply:
+
+```bash
 make mesh
 ```
 
 This will create a binary called `mesh` in ${GOPATH}/bin. Ensure this is in your PATH to run the examples below.
 
-### Flags
+#### Controller
+
+Building a custom controller requires a Dockerhub (or similar) account. To build using the container based build:
+
+```bash
+HUB=docker.io/<your-account> TAG=latest make docker.all
+```
+
+This builds the controller binary and docker file, and pushes the image to the specified hub with the `latest` tag.
+
+Note: The container based build does not work in all setups due to security restrictions. If the above does not work for you,
+try the local build:
+
+```bash
+GOBIN=<your-bin-path> BUILD_WITH_CONTAINER=0 HUB=docker.io/<your-account> TAG=latest make docker.all
+```
+
+Once the images are pushed, configure kubectl to point to your cluster and install the controller. You should edit
+the file deploy/operator.yaml to point to your docker hub:
+
+```yaml
+          image: docker.io/<your-account>/operator
+```
+
+Install the controller manifest:
+
+```bash
+kubectl apply -k deploy/
+```
```

Comment:
missing a step here.  It is also necessary to apply the *cr.yaml file to specify the profile and overrides.

Reply:
Done.